File: beginner/pagina01/1037/1037.cpp
#include <iostream>

using namespace std;

int main(){

	double a;
	cin >> a;

	if(a >= 0 && a <= 25)  cout << "Intervalo [0,25]\n";
	if(a > 25 && a <= 50)  cout << "Intervalo (25,50]\n";
	if(a > 50 && a <= 75)  cout << "Intervalo (50,75]\n";
	if(a > 75 && a <= 100) cout << "Intervalo (75,100]\n";
	if(a > 100 || a < 0)   cout << "Fora de intervalo\n";
	
	return 0;
}

File: beginner/pagina01/1037/1037.lua
local a = tonumber(io.read())

if a >= 0 and a <= 25 then
    print("Intervalo [0,25]")

elseif a > 25 and a <= 50 then
    print("Intervalo (25,50]")

elseif a > 50 and a <= 75 then
    print("Intervalo (50,75]")

elseif a > 75 and a <= 100 then
    print("Intervalo (75,100]")

elseif a > 100 or a < 0 then
    print("Fora de intervalo")
end